CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG
 
+## 18.0.0
```

Comment:
It's a major version bump, so this would indicate that there is a backwards-incompatible change. Could we point out what's the breaking change in this release?